### ISSUE TITLE
het tutorial

### ISSUE THREAD

**Issue by sbellan61** [ICI3D/RTutorials#4]
fix x axes on the ppt slides in ICI3D materials, also make clear that R = beta*D and D=1 on axes of tutorial. clarify that R is as in R0, not recovered

**Comment by dushoff** [comment]
There is a capitalization problem in the load() statement.

On Wed, Jun 1, 2016 at 4:44 AM, Steve Bellan <notifications@github.com>
wrote:

> fix x axes on the ppt slides in ICI3D materials, also make clear that R =
> beta*D and D=1 on axes of tutorial. clarify that R is as in R0, not
> recovered
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ICI3D/RTutorials/issues/4>, or mute the thread
> <https://github.com/notifications/unsubscribe/AC3SebB7RX9vW9uTQzHvj9Ni4Z9LRc1pks5qHUZ9gaJpZM4IrUMZ>
> .
>



-- 
McMaster University Department of Biology
http://lalashan.mcmaster.ca/theobio/DushoffLab/
https://twitter.com/jd_mathbio
http://jd-mathbio.blogspot.com/

**Comment by dushoff** [comment]
Yes, this issue had already been fixed, but it bit me again because it persisted in an obsolete version of the lab (now fixed there, too).
